<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivateCardsDlg</class>
 <widget class="QDialog" name="ActivateCardsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activate cards</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="saved_sets_layout">
       <item>
        <widget class="QLabel" name="saved_sets_label">
         <property name="text">
          <string>Saved sets:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="saved_sets"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Activate cards from these card types:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="active_or_forbidden">
           <item>
            <property name="text">
             <string>having any of these tags:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>not having any of these tags:</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="card_types_tree">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <property name="expandsOnDoubleClick">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTreeWidget" name="tags_tree">
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="ok_button">
           <property name="text">
            <string>&amp;OK</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>&amp;Save this set for later use</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>ActivateCardsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_button</sender>
   <signal>clicked()</signal>
   <receiver>ActivateCardsDlg</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>ActivateCardsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saved_sets</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>ActivateCardsDlg</receiver>
   <slot>load_set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
